crates/core_simd/src/swizzle_dyn.rs

```diff
@@ -0,0 +1,142 @@
+use crate::simd::{LaneCount, Simd, SupportedLaneCount};
+use core::mem;
+
+impl<const N: usize> Simd<u8, N>
+where
+    LaneCount<N>: SupportedLaneCount,
+{
+    /// Swizzle a vector of bytes according to the index vector.
+    /// Indices within range select the appropriate byte.
+    /// Indices "out of bounds" instead select 0.
```

Comment:
Maybe add a note that this really needs `build-std` to work correctly

Reply:
Well, it kinda doesn't, does it? The vectors are generic, so this will get instantiated at compile time. What it needs is to be combined with `target_feature` configuration, either dynamic multiversioning or compile-time versioning or whatnot.

Reply:
The `cfg`s will depend on the features std is built with, unfortunately

Reply:
Oooh good point hmmmmmm...

...I guess I'll make this dynamically multiversioned lol.

Reply:
That part, at least, will be fixed upon promoting this into an intrinsic.